syntax = "proto2";

package Codec_FeeEstimate;

message FeeEstimate
{
	required float feeByte = 1;
	optional bool smartFee = 2;
	optional string error = 3;
}

message FeeSchedule
{
	repeated uint32	     target   = 1;
	repeated FeeEstimate estimate = 2;
}